cmake/FetchLevelZero.cmake

```diff
@@ -40,7 +40,7 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
         set(UR_LEVEL_ZERO_LOADER_REPO "https://github.com/oneapi-src/level-zero.git")
     endif()
     if (UR_LEVEL_ZERO_LOADER_TAG STREQUAL "")
-        set(UR_LEVEL_ZERO_LOADER_TAG v1.18.5)
+        set(UR_LEVEL_ZERO_LOADER_TAG ea296eb72e760b9c04b93e008063ce95cdfe6488)
```

Comment:
How come this is a commit hash but in the fuzz workflow the v0.18.5 tag is used?

I'm Wondering if the discrepancy may cause issues?

Reply:
The fuzz workflow just needed at least v1.18 headers to compile and test, but the version of the loader needed in the Fetch requires a newer version that has several fixes for backwards compatibility.